release_files/install.sh

```diff
@@ -23,19 +23,28 @@ if command -v sudo > /dev/null && [ "$(id -u)" -ne 0 ]; then
     SUDO="sudo"
 fi
 
-get_latest_release() {
+if [ -z ${NETBIRD_RELEASE+x} ]; then 
+    NETBIRD_RELEASE=latest
+fi
+
+get_release() {
+    local RELEASE=$1
+    if [ "$RELEASE" == "latest" ]; then
```

Comment:
```suggestion
    if [ "$RELEASE" = "latest" ]; then
```

Reply:
Sorry for the late reply.
You're right. I only tested my change with bash.
I read through the workflows, and see we test using ubuntu:latest, which default to dash as shell - which in return does not support the "double equal-sign" that bash does.
Sorry about that !

_(I've now imported your validation actions to my fork)_  :smile: 

We should for sure keep the validation action as-is, as it mimic the current recommended one-line installation behavior on the website. _(I should just have tested my change better!)_

This is now corrected to be POSIX compatible. (efc8164bae8325a6cb2937350fde2e268df56dd7)
I imported the validation workflows to my fork - seems to go through now :)